data-reconciliation-app/demo/scripts/02-report_change_demo.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
you need to change the attributes of this file to have execute permission `chmod +x <filename>` and then commit to git

Reply:
Acknowledged.